.github/ISSUE_TEMPLATE/issue_template.yml

```
@@ -0,0 +1,48 @@
+name: Bug Report
+description: Report a bug or unexpected behavior in the Media & Entertainment AI Suite.
+title: "[Bug]: "
+labels: [bug, needs-triage]
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to report a bug. Please fill out the information below to help us address the issue effectively.
+
+  - type: input
+    id: environment
+    attributes:
+      label: Environment
+      description: Describe the environment where the issue occurred (e.g., OS, hardware, software versions).
```

I am not sure how the links show up there though.
```suggestion
      description: |
      Describe the environment where the issue occurred (e.g., OS, hardware, software versions).
      You may utilize the [PerfSpect](https://github.com/intel/PerfSpect) tool to gather system information.
      Please remember not to share any sensitive information.
```